tests/test_declarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -113,6 +117,12 @@ describe(\`does something really long and complicated so I have to write a very
   });
 });
 
+describe("does something really long and complicated so I have to write a very long name for the describe block", willResolve(
+  () => {
```

Comment:
Looks like it puts the arrow function on the next line anyway, so it doesn't actually solve your issue :(